src/relay-utils/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@adeira/relay-utils",
+  "private": false,
+  "version": "0.1.0",
+  "main": "src/index",
+  "module": "src/index.mjs",
+  "sideEffects": false,
+  "homepage": "https://github.com/adeira/universe/tree/master/src/relay-utils",
+  "description": "Relay utils used at Adeira ecosystem",
+  "license": "MIT",
+  "dependencies": {
+    "react-relay": "0.0.0-experimental-5f1cb628",
+    "relay-runtime": "^8.0.0",
+    "@adeira/relay": "0.11.0"
```

Comment:
Isn't `adeira/relay` sufficient enough? 🤔

Reply:
Yes, well, I need relay-runtime, but I don't need react-relay

Reply:
Ah, you mean you are using it directly in the code, right? My thinking was that `adeira/relay` is requiring all these dependencies by default but your approach is better.

Reply:
> Ah, you mean you are using it directly in the code, right? 

Yes, I am importing from `relay-runtime`.

Reply:
@tbergq Shouldn't you specify these as a peer dependency?

Reply:
It could be that instead yes 🤔 👍

Reply:
Updated.